syft/syft.py

```diff
@@ -75,16 +75,22 @@ def __repr__(self):
 
     def __str__(self):
         return self.no_params_func("print",True,False)
-    
+
     def abs(self):
         return self.no_params_func("abs",return_response=True)
-      
+
     def abs_(self):
         return self.no_params_func("abs_")
 
     def neg(self):
         return self.no_params_func("neg")
 
+    def tanh(self):
+        return self.no_params_func("tanh",return_response=False)
```

Comment:
seems like there are a few different naming conventions going on in this file ... e.g. `__function__(params)` vs `function()` ... let me know if I messed it up!